evm_arithmetization/src/testing_utils.rs

```diff
@@ -27,6 +31,44 @@ pub const EMPTY_NODE_HASH: H256 = H256(hex!(
     "56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421"
 ));
 
+/// The recursion threshold when using test configurations
+pub const TEST_THRESHOLD_DEGREE_BITS: usize = 10;
+
+/// The recursion threshold for 2-to-1 block circuit.
+pub const TWO_TO_ONE_BLOCK_CIRCUIT_TEST_THRESHOLD_DEGREE_BITS: usize = 13;
+
+/// A fast STARK config for testing purposes only.
+pub const TEST_STARK_CONFIG: StarkConfig = StarkConfig {
+    security_bits: 1,
+    num_challenges: 1,
+    fri_config: FriConfig {
+        rate_bits: 1,
+        cap_height: 4,
+        proof_of_work_bits: 1,
+        reduction_strategy: FriReductionStrategy::ConstantArityBits(4, 5),
+        num_query_rounds: 1,
+    },
+};
+
+/// A fast Circuit config for testing purposes only.
+pub const TEST_RECURSION_CONFIG: CircuitConfig = CircuitConfig {
+    num_wires: 135,
+    num_routed_wires: 80,
+    num_constants: 2,
+    use_base_arithmetic_gate: true,
+    security_bits: 1,
+    num_challenges: 1,
+    zero_knowledge: false,
+    max_quotient_degree_factor: 8,
+    fri_config: FriConfig {
+        rate_bits: 3,
+        cap_height: 4,
```

Comment:
non-blocking, but is this the fastest test setup? I think increasing the cap size would just increase proof size, and reduce recursive verification? (though this may not matter much as you're adding a shrinking layer by default if the threshold is too small when testing, just wondering)

Reply:
I did not increase the cap_height; it was 4 before.